tests/UsersTest.php

```diff
@@ -152,4 +146,40 @@ public function testNormalUserShouldNotBeAbleDisableMFA()
             $this->assertEquals(403, $e->getCode());
         }
     }
+
+    public function testsRemoveUserFromEverywhere()
+    {
+        $organization = $this->client->createOrganization($this->testMaintainerId, ['name' => 'ToRemoveOrg']);
+        $project = $this->client->createProject($organization['id'], ['name' => 'ToRemoveProj']);
+        $email = 'remove' . uniqid() . '@keboola.com';
+        $this->client->addUserToProject($project['id'], ['email' => $email]);
+        $user = $this->client->getUser($email);
+        $this->client->addUserToOrganization($organization['id'], ['email' => $user['email']]);
+        $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => $user['email']]);
+        $this->client->removeUser($email);
```

Comment:
Snažím se dodržovat http://wiki.c2.com/?ArrangeActAssert (tj. příprava, newline, samotná věc co testuju, newline, asserty - případně i newliny uvnitř těch věcí, když to pomáhá). Takže ten removeUser by měl být oddělený - aby bylo jasné, že je to ta věc, co se testuje.

Reply:
Jo, přesně tak.